keps/sig-node/213-run-as-group/README.md

```diff
@@ -226,14 +226,23 @@ There are other potentially unresolved discussions in that PR which need a follo
 
 ## Graduation Criteria
 
-- Publish Test Results from Master Branch of Cri-o To http://prow.k8s.io [#72253](https://github.com/kubernetes/kubernetes/issues/72253)
-- Containerd and CRI-O tests included in k/k CI [#72287](https://github.com/kubernetes/kubernetes/issues/72287)
-- Make CRI tests failures as release informing
```

Comment:
these were removed, because as per discussion here https://github.com/kubernetes/kubernetes/issues/72253#issuecomment-695147696 , these are not needed and hence a new criteria was added.